internal/pkg/instance/instance_linux_test.go

```diff
@@ -0,0 +1,279 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package instance
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	specs "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/sylabs/singularity/internal/pkg/test"
+)
+
+const testSubDir = "testing"
+
+func TestProcName(t *testing.T) {
+	test.DropPrivilege(t)
+	defer test.ResetPrivilege(t)
+
+	test := ProcName("test", "test")
```

Comment:
here `test` is a new variable; on line 20 `test` is a package selector. To avoid the ambiguity it's probably better to name the variable something else, like `actual` to reflect that this is the actual value obtained from calling the function under test.

Also, without looking at the internals of ProcName, I would expect that there are at least four interesting cases to test:

```
    ProcName("str1", "str2")
    ProcName("str1", "")
    ProcName("", "str1")
    ProcName("", "")
```

- two different strings (to test that the correct one goes in the correct place)
- second string empty
- first string empty
- both strings empty